src/values/dynamic-item-value.js

```diff
@@ -27,6 +33,17 @@ export default class DynamicItemValue {
     return this._metadataJSON;
   }
 
+  get elements() {
+    if (this._elements == null) {
+      this._elements = [];
+      for (const elementJSON of this._elementsJSON) {
+        this._elements.push(ElementFactory.create(null, elementJSON));
```

Comment:
Much better. This way we can also delete this line below:

```
ChildElements.includeInto(DynamicItemValue);
```

Reply:
Done